### ISSUE TITLE
Unable to unpin a pinned cask (wrong number of arguments)

### ISSUE THREAD

**Issue by hkunzhe** [buo/homebrew-cask-upgrade#193]
Hi,
When I try to unpin a pinned cask, an error occurs: `wrong number of arguments (given 2, expected 1)`.
![image](https://user-images.githubusercontent.com/30763967/143543621-8ed82f80-6702-48c1-b8a1-d46c4cef4b62.png)
Does anyone have a suggestion? Thank you in advance!

**Comment by SuperHuangXu** [comment]
me too...

**Comment by cjlittle** [comment]
Same issue over here.  I took a quick look at the codebase, it looks like there is a typo that was introduced in pin_remove.rb (default parameter for quiet was removed in :18 and :26).  

However, running a "manual" upgrade for the cask in question appears to clear the PINNED status.  e.g.:

`brew upgrade siyuan`

I don't have the cycles ATM to get this environment up and running to issue a PR, hopefully someone can get to it before me!
